helm/hpcc/templates/_warnings.tpl

```diff
@@ -86,7 +86,7 @@ Pass in dict with root and warnings
  {{- end -}}
  {{- range $cname, $ctypes := $ctx.components -}}
   {{- range $id, $component := $ctypes -}}
-   {{- if not $component.disabled -}}
+   {{- if and (ne ($id|toString) "disabled") (not $component.disabled) -}}
```

Comment:
I think more correct is:

`   {{- if and (kindIs "map" $component) (not $component.disabled) -}}
`
Otherwise if another property was added to sasha this code would stop working again.